.github/workflows/ci.yaml

```diff
@@ -78,10 +77,11 @@ jobs:
           toolchain: stable
       - uses: actions/setup-python@v2
         with:
-          python-version: ${{ matrix.python-version }}
-      - run: pip install $(grep ^maturin requirements.txt)
-      - if: startsWith(matrix.os,'ubuntu') && matrix.python-version == '3.8'
-        run: maturin publish -u __token__ -p ${{ secrets.MATURIN_PASSWORD }} --interpreter
-        name: Deploy sdist
-      - run: maturin publish -u __token__ -p ${{ secrets.MATURIN_PASSWORD }} -i python --no-sdist
+          python-version: '3.9'
+      - uses: messense/maturin-action@v1
```

Comment:
With the abi3 feature, will this build a single wheel which can be used by all supported pythons, or will it still build different wheels and depend on either the GHA environment or the container environment for which pythons are available?

Reply:
It'll only build a single wheel.

Reply:
And it's manylinux1 compliant.

```
📦 Built wheel for abi3 Python ≥ 3.6 to /home/runner/work/ncollpyde/ncollpyde/target/wheels/ncollpyde-0.15.2-cp36-abi3-manylinux_2_5_x86_64.manylinux1_x86_64.whl
```

Reply:
Aha! That would be great, and probably solve the secondary issues raised in #9 .